atomic_reactor/plugins/pre_add_labels_in_df.py

```diff
@@ -126,6 +131,28 @@ def generate_auto_labels(self, auto_labels):
         docker_info = self.tasker.get_info()
         generated['com.redhat.build-host'] = docker_info['Name']
 
+        info_url = self.info_url_format
+        labels = Labels(self.labels)
+        find_subs = re.finditer('{([^}]+)}', info_url)
+
+        for to_subs in find_subs:
+            found_label = to_subs.group(1)
+            found_type = None
+
+            for label_type, label_list in Labels.LABEL_NAMES.items():
+                if found_label in label_list:
+                    found_type = label_type
+                    break
+
+            if found_type:
+                try:
+                    _, label_value = labels.get_name_and_value(found_type)
+                    info_url = re.sub('{'+ found_label +'}', label_value, info_url)
+                except KeyError:
+                    self.log.warning("label used to construct RHCC url is not defined %s", found_label)
```

Comment:
We shouldn't state its an 'RHCC url' - its just a URL with image info, it might be useful for others who don't use RHCC